new-site/src/docs/elements/components/navigation/customisation.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: '/elements/components/navigation/customisation'
+title: 'Navigation - Customisation'
+---
+
+import { Navigation } from 'hds-react';
+import { Notification } from 'hds-react';
+
+## Customisation
+
+### Customisation properties
+You can use the `theme` property to customise the component. See all available theme variables in the table below.
+
+| Theme variable                           | Description                                                 |
+| ---------------------------------------- | ----------------------------------------------------------- |
+| `--header-background-color`              | Colour of the Navigation background.                        |
+| `--header-color`                         | Colour of elements inside the header row.                   |
+| `--header-divider-color`                 | Colour of the divider between navigation rows.              |
+| `--header-focus-outline-color`           | Colour of the focus outline of the header row elements.     |
+| `--mobile-menu-background-color`         | Colour of the background of an opened mobile menu           |
+| `--mobile-menu-color`                    | Colour of elements inside an opened mobile navigation.      |
+| `--navigation-row-background-color`      | Colour of the navigation (links) row background.            |
+| `--navigation-row-color`                 | Colour of elements inside the navigation row.               |
+| `--navigation-row-focus-outline-color`   | Colour of the focus outline of the navigation row elements. |
+
+
+### Customisation example
+<Playground>
+
+
+```jsx
+<Navigation
+    theme={{
+      '--header-background-color': 'var(--color-bus)',
```

Comment:
Should we change one of these theme variables from default to emphasise that the example can be customised?

Reply:
Probably we need to revise all customisation examples at some point. For now, I just added some basic examples that show all tokens used.